apps/apple/FluentUITester/macos/FluentUITester-macOS/ViewController.m

```diff
@@ -0,0 +1,22 @@
+#import "ViewController.h"
+#import "AppDelegate.h"
+
+#import <React/RCTRootView.h>
+
+@implementation ViewController
+
+- (void)viewDidLoad {
+  [super viewDidLoad];
+
+  RCTBridge *bridge = [((AppDelegate *)[NSApp delegate])bridge];
+  RCTRootView *rootView = [[RCTRootView alloc] initWithBridge:bridge moduleName:@"FluentUITester" initialProperties:nil];
+
+  NSView *view = [self view];
+
+  [view addSubview:rootView];
+  [rootView setBackgroundColor:[NSColor windowBackgroundColor]];

```

Comment:
Is this actually necessary? Do we expect every client to do this?

Reply:
We need this set or the view's transparent. I don't think clients mirroring our test app and setting that is a big issue?